components/raftstore-v2/src/raft/storage.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use engine_traits::RaftEngine;
+use raft::{
+    eraftpb::{Entry, Snapshot},
+    GetEntriesContext, RaftState,
+};
+use slog::Logger;
+
+/// A storage for raft.
+pub struct Storage<ER> {
+    engine: ER,
+    logger: Logger,
+}
+
+impl<ER> Storage<ER> {
```

Comment:
for Raft Storage,  what's the expecting difference between v2 and v1?

Reply:
The initialization is different.